main.go

```diff
@@ -12,10 +12,17 @@ import (
 )
 
 func main() {
-	measurements := make(chan Measurement, 10)
+	rabbit, err := newRabbitMQMeasurementExchanger(os.Getenv("TSM_RABBITMQ_URL"))
```

Comment:
I don't think you should be changing in now, but we usually pass connection to services, etc. Let's say you wanna publish some stuff on some other exchange as well. Doing it the same way you did here, you'd create new `exchanger` - ergo new rabbitmq connection, which is not necessary. But that's minor in this case.